app/services/arcgis_api/geocoder.rb

```diff
@@ -55,7 +60,8 @@ def suggest(text)
       }
 
       parse_suggestions(
-        faraday.get(IdentityConfig.store.arcgis_api_suggest_url, params, dynamic_headers) do |req|
+        connection_factory.connection.
+          get(IdentityConfig.store.arcgis_api_suggest_url, params, dynamic_headers) do |req|
```

Comment:
We no longer use the `.suggest` method, so this method and its tests can be removed instead of updated.